<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanningWidget</class>
 <widget class="QWidget" name="ScanningWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScanningWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mapScaleLabel">
       <property name="text">
        <string>Tỷ lệ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mapScaleComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="regExpLabel">
       <property name="text">
        <string>Biểu thức luật:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="regExpLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="scanningDirLabel">
       <property name="text">
        <string>Thư mục cần quét:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scanDirLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseScanDirButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Chọn...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scanButton">
       <property name="text">
        <string>Quét</string>
       </property>
       <property name="icon">
        <iconset resource="../MainWindow.qrc">
         <normaloff>:/icons/scan.svg</normaloff>:/icons/scan.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string>Số lượng: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
